client/gutenberg/extensions/gif/index.js

```diff
@@ -8,8 +8,10 @@ import { Path, SVG } from '@wordpress/components';
  * Internal dependencies
  */
 import edit from './edit';
-import './editor.scss';
+
+// Ordering is important! Editor overrides style!
 import './style.scss';
+import './editor.scss';
```

Comment:
I don't think this change is necessary now that the overflow and overflow override have been removed.